curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/6610bbed59bc2a0194d85533.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 6610bbed59bc2a0194d85533
+title: Step 19
+challengeType: 0
+dashedName: step-19
+---
+
+# --description--
+
+Notice how the value inside your `rows` array has been changed directly? This is called <dfn>mutation</dfn>. As you learn more about arrays, you will learn when to mutate an array, and when you should not.
+
+Before moving on, this is a great opportunity to learn a common array use. Currently, your code accesses the last element in the array with `rows[2]`. But you may not know how many elements are in an array when you want the last one.
+
+You can make use of the `.length` property of an array - this returns the number of elements in the array. For example, your `rows` array has 3 elements, so `rows.length` would be `3`.
```

Comment:
missing backticks for number 3

```suggestion
You can make use of the `.length` property of an array - this returns the number of elements in the array. For example, your `rows` array has `3` elements, so `rows.length` would be `3`.
```

Reply:
That's not referring to the code, though. "Your thing has three elements".

The second 3 refers to the code.